pkg/terraform.go

```diff
@@ -199,6 +200,22 @@ func (e *Executor) showRaw(dir string, tfBinaryLocation string) (string, error)
 	return out, err
 }
 
+// our jenkins instances are set to time out after 30 minutes of no logs to the console so this simply
+// prints out a log message once a minute that the apply/plan/destroy is still in progress so pipelines don't time out
```

Comment:
terraform should print out in progress logs, can we stream it out?

Reply:
You're completely right! I'm not sure why we were printing logs only at the end but I can fix that.